packages/formation/sass/base/_b-variables.scss

```diff
@@ -90,6 +90,17 @@ $green-warm-50: #6f7a41;
 $blue-50: #357ab2;
 $violet-warm-60: #864381;
 
+// Contextual variable names
+$color-hub-healthcare: $indigo-cool-60;
+$color-hub-education: $mint-cool-50;
+$color-hub-disability: $red-60;
+$color-hub-careers: $orange-warm-50;
+$color-hub-pension: $green-cool-50;
+$color-hub-housing: $gold-50;
+$color-hub-life-insurance: $green-warm-50;
+$color-hub-burials: $blue-50;
+$color-hub-records: $violet-warm-60;
```

Comment:
I think we just added some more hubs in vets-website (see src/platform/site-wide/sass/iconography.scss). Should we add variables for those, too?

Reply:
Oh, did not know. Yes, we should! I see they are the same color, so I'm just going to check with the designers to confirm

Reply:
Added new hubs.